ubyssey/templates/homepage/base.html

```diff
@@ -44,13 +44,7 @@ <h2 class="headline"><a href="{{ article.get_absolute_url }}?ref=frontpage">{{ a
                 </article>
                 {% endwith %}
             </div>
-            <div class="multi-zone cell right">
-                {% if day_of_week == 5 or day_of_week == 6 %}
-                    {{ components.multi_zone|safe }}
-                {% else %}
-                    {{ components.print_issue|safe }}
```

Comment:
We actually want to keep this line:

`{{ components.print_issue|safe }}`

Sorry, should have been more clear in the issue. You can remove the `if/else` and just keep the `print_issue` line.